src/components/card/mod.rs

```diff
@@ -91,49 +85,48 @@ pub struct CardProperties {
 /// ```
 #[function_component(Card)]
 pub fn card(props: &CardProperties) -> Html {
-    let expanded = use_state_eq(|| false);
+    let expanded = use_state_eq(|| !props.expandable);
 
     let mut class = classes!("pf-v5-c-card");
 
     if props.compact {
-        class.push(classes!("pf-m-compact"));
+        class.push("pf-m-compact");
+    }
+
+    if props.disabled {
+        class.push("pf-m-disabled");
     }
 
     if props.expandable && *expanded {
-        class.push(classes!("pf-m-expanded"));
```

Comment:
To my understanding the `classes!` macro does splitting during build time, while using a string performs splitting during runtime.

If that's the case, then I think keeping the classes macro does save a bit of performance.

Reply:
I think I looked at the wrong revision, please ignore that.